maintainers/maintainer-list.nix

```diff
@@ -9363,6 +9363,16 @@
     githubId = 1391883;
     name = "Tom Hall";
   };
+  Thunderbottom = {
```

Comment:
Can you split this into a separate commit with a message like 'maintainers: add Thunderbottom'

Reply:
this is done, thanks for spotting!